app/views/eventDetails/paymentOptions.html

```diff
@@ -61,57 +71,58 @@ <h3 class="panel-title" translate>Checks</h3>
       </div>
       <div class="col-sm-3 stacked-spacing-col-sm">
         <select class="form-control" ng-model="conference.checkMailingState">
-          <option value="AL">Alabama</option>
```

Comment:
From the other places it is used, it looks like putting the translate attribute on an element translates the text content of that element. Since it is the text content of the option elements that we want to translate, I think you are good to do it that way.